.github/workflows/format-and-commit.yml

```
@@ -0,0 +1,39 @@
+name: Format And Commit
+
+concurrency:
+  group: ${{ github.ref }}
+
+on:
+    # Pending getting through individual PRs.
+    # push:
+    #   branches:
+    #     - master
+    workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  dprint-fmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        check-latest: true
+        node-version: '*'
```

```suggestion
        node-version: '16'
```
should this one be in sync with others? e.g.:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.github/workflows/CI.yml#L15